zio-http/src/it/scala/zhttp/IntegrationSpec.scala

```diff
@@ -0,0 +1,65 @@
+package zhttp
+
+import zhttp.http.HttpData.CompleteData
+import zhttp.http.Response.HttpResponse
+import zhttp.http._
+import zhttp.service._
+import zio._
+import zio.duration._
+import zio.test.Assertion._
+import zio.test.TestAspect.timeout
+import zio.test._
+
+object IntegrationSpec extends DefaultRunnableSpec {
+  import IntegrationSpecHelper._
+
+  def spec = suite("IntegrationSpec")(
+    ApiSpec,
+  ).provideCustomLayer(env) @@ timeout(10 seconds)
+
+  def ApiSpec = suite("ApiSpec") {
+    testM("200 ok on /") {
+      val zResponse = Client.request(baseAddr)
+      assertM(zResponse.map(_.headerRemoved("date")))(
```

Comment:
```suggestion
      assertM(zResponse)(
```

We can do this and make servertime configurable